src/components/types.ts

```diff
@@ -272,3 +272,12 @@ export interface QuestionFormat extends DataFormat {
 }
 
 export interface QuestionProps extends QuestionFormat, ConversationProps {}
+
+export interface PDFViewerProps {
+  /** The url of the PDF file to be displayed. */
+  url: string
+  /** Used to decide if the PDF viewer should be displayed. */
+  isOpen: boolean
+  /** Function to handle closing the PDF viewer. */
+  onClose: () => void
```

Comment:
If the viewer is going to open in a modal, do we still need open and close functions ?

Reply:
Since an open button is not included in the PDF viewer, we need a way to know when to open the viewer. I also added a delete button so users don't have to close it by clicking on the grey area outside the viewer.